packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -167,11 +171,13 @@ export function PrivateBlockToolbar( {
 	// Shifts the toolbar to make room for the parent block selector.
 	const classes = clsx( 'block-editor-block-contextual-toolbar', {
 		'has-parent': showParentSelector,
+		'is-dark-toolbar': ! isDefaultEditingMode && ! hasFixedToolbar,
```

Comment:
I'd avoid words that dictate the appearance (`dark`) as we may need to re-evaluate the design when we introduce light/dark modes down the road.

Reply:
So how you would name this. See my previous comment here https://github.com/WordPress/gutenberg/pull/66116#discussion_r1810685860

Reply:
We could use `inverted` or `alt` (as in `alternative`)?

Reply:
@dhruvang21 I think this line causes the issues in my video - `isDefaultEditingMode` applies to indivudual blocks which can be locked or in partially synced patterns so we can end up with the dark toolbar in default mode.